src/event/attributes.rs

```diff
@@ -68,6 +70,22 @@ pub(crate) trait DataAttributesWriter {
     fn set_dataschema(&mut self, dataschema: Option<impl Into<Url>>);
 }
 
+#[derive(PartialEq, Debug, Clone, Copy)]
+pub enum AttributesIter<'a> {
```

Comment:
do we need this to be pub? Does it work with `pub(crate)`?

Reply:
yupp